roles/keycloak/tasks/configure.yml

```diff
@@ -139,6 +139,7 @@
         id: "{{ item.name }}"
         realm: "{{ item.name }}"
         login_theme: "{{ keycloak_plugins.wayf.theme.name }}"
+        connection_timeout: 60
```

Comment:
Is this required?

Reply:
We could drop this, but in heavy loaded setups, it might fail to execute (default timeout is 10sec, which is very low). 
In case of a failure at that point, we could just execute the `"keycloak:config:wayf_plugin"` tag which would succeed (i believe, because in this case no other previous task would would keep it busy).
Let me see if we can just rewrite this not to use this extension. 

I also found a small bug here: since the day i created this PR, this theme is also used for the user account. So, we are missing an extra option to set this theme as default for the account.
![scr1](https://user-images.githubusercontent.com/11722418/162903942-8569cdd1-1ea9-4604-b0f4-8f60f63db168.png)
Let me set it as 'draft' till i fix all the above.

Reply:
Updated the code 
- not to use the plugin
- to select the theme for the account console (fixed the bug)